worlds/tunic/__init__.py

```diff
@@ -110,6 +110,13 @@ class TunicWorld(World):
     ut_can_gen_without_yaml = True  # class var that tells it to ignore the player yaml
 
     def generate_early(self) -> None:
+        try:
+            int(self.settings.disable_local_spoiler)
+        except AttributeError:
+            raise Exception("You have a TUNIC APWorld in your lib/worlds folder and custom_worlds folder.\n"
+                            "This would cause an error at the end of generation.\n"
+                            "Please remove one of them, preferably the one in lib/worlds.")
```

Comment:
"Preferably" feels a little weird here, it's not really a preference, just one that's more likely to be correct
```suggestion
                            "Please remove one of them, most likely the one in lib/worlds.")
```